VA-Explorer/va_explorer
# added location to dashboard header and fixed map loading bug

This PR addresses two issues: 
1. Added location details to dashboard header
2. Fixed bug preventing map from loading for district-level location restrictions

To test header, simply switch to dashboard and confirm that header updates with chosen region each time you zoom. By default, it should say "for All Regions"

To test location bug is fixed: 
1. Log in as admin
2. Create a data manager/viewer tied to a certain district that you know has VAs
3. Login as this user and browse to dashboard
4. confirm that map loads and displays region of interest. 

Closes #116, Closes #137
